src/containerapp/azext_containerapp/custom.py

```diff
@@ -603,6 +605,7 @@ def update_containerapp(cmd,
                         remove_all_env_vars=False,
                         cpu=None,
                         memory=None,
+                        gpu=None,
```

Comment:
For the new added params, you need to set them to the payload. (For example: https://github.com/Azure/azure-cli-extensions/blob/70983a7e17860f575a5f94d581d00bd4f61e7ab4/src/containerapp/azext_containerapp/containerapp_decorator.py#L1282)